### PR TITLE
Some typo fixes, suggestions, comments

### PR DESCRIPTION
**Pull request by david-mears-2**
Excellent walk-through of the reasoning process.

Nice use of footnotes. Do we know if they work on the blog site (i.e. they result in hyperlinks)? I checked if there were already some in use there, and didn't see any.

I was finding it hard to follow some of the discussion of R's garbage collector because I don't know what R_VSize is.

### CODE REVIEW

**Comment by plietar** [comment]
> Do we know if they work on the blog site (i.e. they result in hyperlinks)? I checked if there were already some in use there, and didn't see any.

I haven't actually checked, but I assume they work. It's a standard markdown feature.

> I was finding it hard to follow some of the discussion of R's garbage collector because I don't know what R_VSize is.

Yeah you're right, I do bring it up without explanation. Will clarify.

**Comment by EmmaLRussell** [comment]
Piggy-backing my comments on David's PR...

Great explanation of GC, and the visualisations are really effective at helping to understand what's going on. 

> `Garbage collection 3306 = 2388+410+508 (level 2) ...`

What does the "level n" label at the end of the garbage collection summary line refer to? The level of the last collection that was done... or something else?

> That is, if we've just collected NUM_OLD_GENERATIONS (ie. a level-2 collection), then try to adjust the heap size by calling the AdjustHeapSize function. This function tries to determine whether R_VSize should be increased, and by how much. Below is a reproduction of the relevant lines from that function.

Does this mean that, because it's done as much as it can with the existing R_VSize by gc'ing up to level 2, at this point it resorts to working out if it can/should adjust R_VSize? And it's going to be doing this a lot because of how much level 2 gc'ing is going on?

Does it call `AdjustHeapSize` on every level 2 collection, including the 1-in-100 scheduled ones, or only on the 'emergency' ones that we escalated to because of high memory usage?

I haven't attempted to parse the logic in `AdjustHeapSize` as it sounds like that's probably going to be the subject of Part 2!
